src/SourceGenerators/Uno.UI.SourceGenerators/DependencyObject/DependencyObjectGenerator.cs

```diff
@@ -890,6 +894,16 @@ private static void GenerateDependencyObjectImplementation(IndentedStringBuilder
 				builder.AppendLineInvariant("public long RegisterPropertyChangedCallback(DependencyProperty dp, DependencyPropertyChangedCallback callback) => __Store.RegisterPropertyChangedCallback(dp, callback);");
 
 				builder.AppendLineInvariant("public void UnregisterPropertyChangedCallback(DependencyProperty dp, long token) => __Store.UnregisterPropertyChangedCallback(dp, token);");
+
+				if (_isUnoSolution && !typeSymbol.IsSealed)
+				{
+					builder.AppendLineInvariant("void IDependencyObjectInternal.OnPropertyChanged2(global::Windows.UI.Xaml.DependencyPropertyChangedEventArgs args) => OnPropertyChanged2(args);");
+
+					if (typeSymbol.GetMethodsWithName("OnPropertyChanged2").None(m => m.Parameters.Length == 1))
```

Comment:
You could also check the type of parameter :)

Reply:
It's inside of uno, it'll be easy to troubleshoot.